### DISCUSSION TITLE
Stellarium on Kindle

### DISCUSSION THREAD

**Discussion by Steve-Horndean** [Stellarium/stellarium#1751] [General]
I had Stellarium on my Kindle HD10, but it is not in the app store for Kindle HD10+

any idea why ?

**Comment by xalioth** [comment]
If you mean Stellarium Mobile, it should still be available on Kindle, but you should maybe check the version that can be installed from Google Play instead ?

**Comment by Steve-Horndean** [comment]
Thanks for the reply Fabien.

 

I still have my Kindle HD10, and Stellarium is still available from the Amazon Appstore. It is not available from the Appstore on my HD10+. It’s possibly something that Amazon have done ?

 

I cannot install the Google play onto kindle. Again, I think Amazon have blocked anything ‘Google’ from their fire OS.

 

Thanks for your assistance.

 

Steve Raines

**Comment by xalioth** [comment]
OK, then I don't really know the reason. We somewhat stopped supporting the amazon platform and didn't update the app since several years, so it's quite likely that they introduced mandatory requirements for newer devices, like maybe 64 bits support which we didn't have back then.

We'll have a look at that.

**Comment by Steve-Horndean** [comment]
Thanks Fabien, I'll continue to use Stellarium on my older Kindle.. It's a
great program !

 

Regards


Steve
